src/optimization/iLP.jl

```diff
@@ -44,30 +44,41 @@ function solve(solver::ILP, problem::Problem)
     if !solver.iterative
         encode_network!(model, nnet, neurons, δ, StandardLP())
         optimize!(model)
-        termination_status(model) != OPTIMAL && return AdversarialResult(:unknown)
-        return interpret_result(solver, value(o), problem.input)
+        J = value(o)
+        if isfeasible(model)
```

Comment:
Upon second review, `iLP` seems to be safe from this issue. Its objective is minimizing the infinity-norm, so is naturally bounded.